rest/api.go

```diff
@@ -54,6 +55,7 @@ func (h *handler) handleRoot() error {
 		Vendor: vendor{
 			Name: base.ProductNameString,
 		},
+		PersistentConfig: h.server.persistentConfig,
```

Comment:
Can we update the API documentation in `docs/api/paths/admin/~.yaml` and `docs/api/paths/public/~.yaml` to reflect this change?

The response schema seems to be copied across both files for some reason. So if you want to do a bit of a clean up job, you can move the response schema into a separate file (`docs/api/components/schemas.yaml`) and reference it from both `~.yaml` files like most of the other response bodies.